invoice/src/test/java/org/killbill/billing/invoice/dao/TestCBADaoUnit.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020-2022 Equinix, Inc
+ * Copyright 2014-2022 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.invoice.dao;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import org.killbill.billing.invoice.InvoiceTestSuiteNoDB;
+import org.mockito.Mockito;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * Add "Unit" in the end of class name to indicate that this is unit/fast test.
```

Comment:
FWIW All test classes that inherit from `*NoDB` are `fast` (aka unit) tests.

Reply:
Ok, comment removed. But keep "Unit" word as we probably want to have dedicated `TestCBADao` (integration) in the future.